app/controllers/sign_up/completions_controller.rb

```diff
@@ -91,6 +91,19 @@ def analytics_attributes(page_occurence)
         in_account_creation_flow: user_session[:in_account_creation_flow] || false,
         needs_completion_screen_reason: needs_completion_screen_reason,
       }
+
+      if page_occurence.present? && DisposableDomain.disposable?(email_domain)
```

Comment:
What is the purpose of the `page_occurence.present?` check here? Can we add test coverage if it's significant?

Reply:
@aduth The function `analytics_attributes` generates the payload for the events here. The testing for this is captured in the varied outputs of payloads for different events. For the `track_completion_event`, this event is only called from "named pages".

```
user_registration_agency_handoff_page_visit(**analytics_attributes('')
user_registration_complete(**analytics_attributes(last_page))
```

Reply:
Ok, in that case can we add a test for `#show` action that asserts that a user with a disposable email address would not have the `:disposable_email_domain` logging detail included in `user_registration_agency_handoff_page_visit` ?